test/integration/server-side-dev-errors/test/index.test.js

```diff
@@ -202,63 +318,395 @@ describe('server-side dev errors', () => {
+           6 |   setTimeout(() => {
+        >  7 |     Promise.reject(new Error('catch this rejection'))
+             |                    ^
+           8 |   }, 10)
+           9 |   return {
+          10 |     props: {},
+        "
+      `)
+    }
   })
 
   it('should show server-side error for uncaught empty rejection correctly', async () => {
     const stderrIdx = stderr.length
     await webdriver(appPort, '/uncaught-empty-rejection')
 
-    await check(async () => {
-      const cleanStderr = stripAnsi(stderr.slice(stderrIdx))
+    await retry(() => {
+      expect(stderr.slice(stderrIdx)).toContain('Error:')
+    })
 
-      return cleanStderr.includes('pages/uncaught-empty-rejection.js') &&
-        cleanStderr.includes('7:20') &&
-        cleanStderr.includes('getServerSideProps') &&
-        cleanStderr.includes('new Error()')
-        ? 'success'
-        : cleanStderr
-    }, 'success')
+    const stderrOutput = stripAnsi(stderr.slice(stderrIdx))
+    // FIXME(veil): error repeated
+    // FIXME(veil): codeframe repeated after " ⨯ unhandledRejection: Error:"
+    if (isTurbopack) {
+      // TODO(veil): digest: undefined should be omitted?
+      expect(stderrOutput).toMatchInlineSnapshot(`
+        "Error: 
+            at Timeout._onTimeout (../../test/integration/server-side-dev-errors/test/integration/server-side-dev-errors/pages/uncaught-empty-rejection.js:7:19)
+           5 | export async function getServerSideProps() {
+           6 |   setTimeout(() => {
+        >  7 |     Promise.reject(new Error())
+             |                   ^
+           8 |   }, 10)
+           9 |   return {
+          10 |     props: {},
+         ⨯ test/integration/server-side-dev-errors/pages/uncaught-empty-rejection.js (7:20) @ Timeout._onTimeout
+         ⨯ test/integration/server-side-dev-errors/pages/uncaught-empty-rejection.js (7:20) @ Timeout._onTimeout
+         ⨯ unhandledRejection: Error: 
+            at Timeout._onTimeout (./test/integration/server-side-dev-errors/pages/uncaught-empty-rejection.js:7:20) {
+          digest: undefined
+        }
+           5 | export async function getServerSideProps() {
+           6 |   setTimeout(() => {
+        >  7 |     Promise.reject(new Error())
+             |                    ^
+           8 |   }, 10)
+           9 |   return {
+          10 |     props: {},
+         ⨯ unhandledRejection: Error: 
+            at Timeout._onTimeout (./test/integration/server-side-dev-errors/pages/uncaught-empty-rejection.js:7:20) {
+          digest: undefined
+        }
+           5 | export async function getServerSideProps() {
+           6 |   setTimeout(() => {
+        >  7 |     Promise.reject(new Error())
+             |                    ^
+           8 |   }, 10)
+           9 |   return {
+          10 |     props: {},
+        "
+      `)
+    } else {
+      expect(stderrOutput).toMatchInlineSnapshot(`
+              "Error: 
```

Comment:
is this the same case where we potentially can have less indentation?

Reply:
This highlights why the added intendation is bad. Userland `console.error(error)` will be misaligned compared to the `logErrorWithOriginalStack` calls.